src/components/pages/reusable-ui/Input.js

```diff
@@ -1,24 +1,81 @@
-import styled from 'styled-components'
-import { theme } from '../../../theme'
+import { useRef } from "react";
+import styled, { css } from "styled-components";
+import { theme } from "../../../theme";
 
-export default function Input({value,onChange,Icon, ...extraProps}) {
-
-    return (
-        <InputStyled>
-            <span className="icon">{Icon && Icon}</span>
-            <input
-                value={value}
-                onChange={onChange}
-                {...extraProps}
-            />
-        </InputStyled>
-    )
+export default function Input({
```

Comment:
```suggestion
export default function TextInput({
```

Je conseille de l'appeler TextInput car tu as mis en type="text" et par la suite, on aura SelectInput et peut etre même après CheckboxInput